app/templates/components/announcement-box.hbs

```diff
@@ -1,14 +1,18 @@
-<div
-  class="button expanded tiny text-left no-margin"
-  style="text-align: left;"
-  onclick={{action this.toggle}}
->
-  {{fa-icon 'bullhorn'}}
-  Announcement: {{this.title}}
-</div>
-
-{{#if this.isOpen}}
-  <div class="announcement-box" style="background-color: #ecedee">
-    {{yield}}
-  </div>
-{{/if}}
+<button>
+  {{fa-icon "bell"}} <sup class="a11y-orange">NEW!</sup>
+  <EmberPopover
+    @side="bottom-end"
+    @event='click'
+    @tooltipClassName='ember-popover dark'
+    @popperContainer='body'
+    as |popover|
+  >
+    <p class="text-small">
+      We've been working on ZoLa to prevent service interruptions. With these upgrades, you should now notice performance improvement.
+    </p>
+    <p class="text-small">
+      Thank you for using ZoLa!
+      <br>—<a href="mailto:zola_gis@planning.nyc.gov"> NYC Planning Labs</a>
+    </p>
```

Comment:
Looks great! Why keep this inside the component instead of passing it as a block from the invoking side?

Reply:
That would work. I had the whole thing in the site header, without using a component at all. I hastily moved it into the component. Makes sense to use it as a block component with the content in the header.